docs/django-admin/roles.md

```diff
@@ -1,21 +1,16 @@
 # Django admin user roles
 
-Roles other than superuser should be defined in authentication and authorization groups in django admin 
+For our MVP, we create and maintain 2 admin roles:
+Full access and CISA analyst. Both have the role `staff`.
+Permissions on these roles are set through groups:
+`full_access_group` and `cisa_analysts_group`. These
```

Comment:
A bit like how it was done previously in the now deleted "Superuser" section, I think we should roughly define where cisa_analysts_group is restricted.  I don't think it needs to be super detailed but having a reference here would be useful if we need to make a change. Could even be as simple as redirecting them to look at 
user_group for more context